src/librbd/ImageCtx.h

```diff
@@ -202,7 +202,7 @@ namespace librbd {
 
     typedef boost::lockfree::queue<
       io::AioCompletion*,
-      boost::lockfree::allocator<ceph::allocator<void>>> Completions;
+      boost::lockfree::allocator<ceph::allocator<void*>>> Completions;
```

Comment:
This line is two years old.  Has it been bugging you for two years or something else changed?

Reply:
> 
> 
> This line is two years old. Has it been bugging you for two years or something else changed?

Not sure if it is for 2 year, but for quite some time. And it is very likely that libc++ has become (more) picky in this case as well. I have some more time to spare and want to make some FreeBSD ports releases, so I felt like cleaning up.